src/utils/index.js

```diff
@@ -74,3 +74,36 @@ export function getGitValues(url) {
 
   return { gitServer, gitOrg, gitRepo: `${gitRepo}.git` };
 }
+
+// Update the status of steps that follow a step with an error
+export function updateUnexecutedSteps(steps) {
+  if (typeof steps === 'undefined') {
```

Comment:
is there any change `steps` could be null? if so might be safer just to check `if (!steps) {`

Reply:
Ok, thanks. I changed it. 

How about for these lines (https://github.com/tektoncd/dashboard/pull/653/files#diff-44384c8c75c7b5aab1477fe8f0736133R162-R163):
```
typeof step.stepStatus !== 'undefined' &&
typeof step.stepStatus.terminated !== 'undefined'
```
@AlanGreene Is it safer to do `if (!step.stepStatus && step.stepStatus.terminated)...` ?

Reply:
`if (step.stepStatus && step.stepStatus.terminated) {`

yes, easier to read / reason about too (IMO)

Reply:
Ok, thanks! 👍